cloudinit/distros/freebsd.py

```diff
@@ -25,6 +31,8 @@ class Distro(cloudinit.distros.bsd.BSD):
     pkg_cmd_install_prefix = ["pkg", "install"]
     pkg_cmd_remove_prefix = ["pkg", "remove"]
     pkg_cmd_update_prefix = ["pkg", "update"]
+    pkg_cmd_upgrade_prefix = ["pkg", "upgrade"]
+    home_dir = '/usr/home'
```

Comment:
or this, for that matter…

Reply:
Oops, the line was coming from https://github.com/canonical/cloud-init/pull/305